### PR TITLE
Desktop site

### PR DESCRIPTION
**Pull request by badger-99**
In this Milestone I have:
- Introduced a responsive desktop version of the website with breakpoints at 768px and 1200px.
- Resolved the [issue](https://github.com/badger-99/Portfolio/issues/4) brought up from the code-review of my previous pull request.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -376,3 +392,287 @@ hr {
...
+
+  /* Project Grid */
+  #projects_grid {
+    margin: 8 auto 8.125rem auto;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr fr;
+    gap: 2.313rem 1.5rem;
+  }
+
```
Comment (Johnadibe) [inline styles.css:521]: > - [x] Good work @badger-99👍. Kindly adjust the padding of each grid card to stick to the Figma design as much as possible.☺
> 
> Figma design👇:
> 
> padding-figma
> 
> Your design👇:
> 
> padding
> 
> 
>

File: styles.css [hunk 2]
```diff
@@ -376,3 +392,287 @@ hr {
...
+    padding-left: 8.875rem;
+    padding-right: 45.75rem;
+    margin: 0;
+  }
+
+  #about_text {
+    padding-left: 8.875rem;
+    margin-right: 45.75rem;
+  }
+
```
Comment (Johnadibe) [inline styles.css]: > - [x] Keep doing great work👍 I noticed that your text is clustered together. Kindly include a `line-height` property to your CSS code to stick with the Figma design as much as possible. Setting a line-height value across your website helps to create a consistent visual style that makes your website more visually appealing and professional.☺
> 
> Figma design👇:
> 
> line-height-figma
> 
> Your design👇:
> 
> line-height
> 
>

File: styles.css [hunk 3]
```diff
@@ -376,3 +392,287 @@ hr {
...
+  /* Abilities */
+
+  #abilities {
+    padding: 0;
+    margin: 0;
+    padding-top: 8.625rem;
+  }
+
```
Comment (Johnadibe) [inline styles.css:564]: > - [x] Well done👍 I suggest you align your technology categories to stick with the Figma design as much as possible.  Sticking with the Figma design ensures that the typography, layout, and visual design are consistent throughout the entire project, which can improve the overall user experience.☺
> 
> Figma design👍:
> 
> left-figma
> 
> Your design👇:
> 
> left
>

File: styles.css [hunk 4]
```diff
@@ -125,14 +140,14 @@
   margin-bottom: 1.5rem;
 }
```
Comment (Johnadibe) [inline styles.css:142]: > [OPTIONAL] While building the desktop version of your project, I noticed that the margin-bottom spacing overlaps with the project card container on a mobile screen. Please kindly fix that☺
> 
> Figma design👇:
> 
> optional-figma
> 
> Your design👇:
> 
> optional
>

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @badger-99👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You used GitHub Flow correctly 👍🏽
✅ Linter checks passed✔️
✅ Used Grid and Flexbox to lay out elements 👍🏽
✅ Used media queries for the Desktop breakpoint✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @badger-99 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**